packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -732,6 +757,13 @@ const Dashboard: FC = () => {
                         onConfirm={duplicateModalHandlers.close}
                     />
                 )}
+                {isFilterWarningModalOpen && (
+                    <DashboardFiltersWarningModal
+                        onConfirm={handleEnterEditMode}
+                        onClose={() => setIsFilterWarningModalOpen(false)}
+                        opened={isFilterWarningModalOpen}
+                    />
+                )}
```

Comment:
```suggestion

                    <DashboardFiltersWarningModal
                        onConfirm={handleEnterEditMode}
                        onClose={() => setIsFilterWarningModalOpen(false)}
                        opened={isFilterWarningModalOpen}
                    />
```

NP: this is sufficient

Reply:
Will do. We often follow that existing pattern to intentionally mount and unmount the modal, but this one has no state so it doesn't matter.

Reply:
Actually I'm gonna keep this for consistency with all of the other modals on this page, if that's cool. There are a lot of them. And un-mounting the unopened ones seems ok.

Reply:
makes sense!